docker-compose.yml

```diff
@@ -5,10 +5,17 @@ services:
       context: ./
       args:
         BUILD-FLAVOUR: ${BUILD-FLAVOUR:-Release}
+    environements:
+      - DB_HOST=Server
```

Comment:
Should this be taken from .env file?

Reply:
not really the goal is not to use this compose file, it is more an example